doc/sphinx/source/recipes/recipe_iht_toa.rst

```diff
@@ -0,0 +1,123 @@
+.. _recipes_iht_toa:
+
+Implied heat transport from Top of Atmosphere fluxes
+====================================================
+
+Overview
+--------
+
+This recipe calculates the implied horizontal heat transport (IHT) due to the
+spatial anomalies of radiative fluxes at the top of the atmosphere (TOA).
+The regional patterns of implied heat transport for different components of
+the TOA fluxes are calculated by solving the Poisson equation with the flux
+components as source terms.
+It reproduces the plots in `Pearce and Bodas-Salcedo (2023)`_ when the input
+data is CERES EBAF.
+
+
+Available recipes and diagnostics
+---------------------------------
+
+Recipes are stored in esmvaltool/recipes/
+
+    * recipe_iht_toa.yml calculates the IHT maps for the following radiative fluxes:
+
+        * Total net, SW net, LW net (Figure 2).
+        * Total CRE, SW CRE, LW CRE (Figure 4).
+        * All-sky and clear-sky reflected SW (Figure 5).
+        * The meridional heat transports (MHT) of the fluxes above (Figures 1 and 3).
+
+Diagnostics are stored in esmvaltool/diag_scripts/iht_toa/
+
+    * single_model_diagnostics.py: driver script that produces the plots.
+    * poisson_solver.py: solver that calculates the IHTs.
+
+.. _`Pearce and Bodas-Salcedo (2023)`:  https://doi.org/10.1175/JCLI-D-22-0149.1
+
+User settings in recipe
+-----------------------
+
```

Comment:
If there are no user settings in the recipe, it is better to say "There are no user settings in this recipe" or equivalent, rather than leaving this section empty :)

Reply:
Changed as suggested.